<launch>
    <node pkg="team2_package" type="filter_ver_2" name="filter_ver_2">
        <param name="roi_upper" value="0.1"/>
        <param name="roi_lower" value="-1.25"/>
        <param name="roi_front" value="2.0"/>
        <param name="roi_back" value="-2.0"/>
        <param name="roi_left" value="-1.0"/>
        <param name="roi_right" value="1.0"/>
        <param name="ransac_threshold" value="0.7"/>
        <param name="leaf_size_x" value="0.15"/>
        <param name="leaf_size_y" value="0.15"/>
        <param name="leaf_size_z" value="0.15"/>

        <param name="min_cluster_vol" value="3"/>
        <param name="max_cluster_vol" value="200"/>
        <param name="max_cluster_dist" value="20"/>
        <param name="eps" value="0.7"/>
        <param name="minPts" value="10"/>
        <param name="maxPts" value="100"/>
        <param name="bbox_min" value="0.01"/>
        <param name="bbox_max" value="10"/>
        <param name="loop_rate_" value="1"/>

    </node>
</launch>